src/components/AssignAttributeDialog/AssignAttributeDialog.tsx

```diff
@@ -25,7 +24,7 @@ import useModalDialogErrors from "@saleor/hooks/useModalDialogErrors";
 import useModalDialogOpen from "@saleor/hooks/useModalDialogOpen";
 import useSearchQuery from "@saleor/hooks/useSearchQuery";
 import { buttonMessages } from "@saleor/intl";
-import { makeStyles } from "@saleor/macaw-ui";
+import { Button, makeStyles } from "@saleor/macaw-ui";
```

Comment:
We're changing the import but the color/variant remains the same. Is the new default <Button /> equivalent to the old one?

Reply:
It is